src/serverdlg.cpp

```diff
@@ -353,6 +352,8 @@ lvwClients->setMinimumHeight ( 140 );
     // update GUI dependencies
     UpdateGUIDependencies();
 
+    UpdateRecorderStatus ( QString::null );
```

Comment:
For compatibility with both Qt5 and Qt6, this should be `QString()` instead of `QString::null`.

Reply:
This is just moving code...  There would be other places:
```
src/serverdlg.cpp:    UpdateRecorderStatus ( QString::null );
src/serverdlg.cpp:    UpdateRecorderStatus ( QString::null );
src/serverdlg.cpp:    UpdateRecorderStatus ( QString::null );
src/serverdlg.cpp:    UpdateRecorderStatus ( QString::null );
src/serverdlg.cpp:    if ( pServer->GetRecorderErrMsg() != QString::null )
src/serverdlg.cpp:        UpdateRecorderStatus ( QString::null );
src/serverdlg.cpp:    if ( pServer->GetRecorderErrMsg() != QString::null || pServer->GetRecordingDir() != "" )
src/serverdlg.cpp:        UpdateRecorderStatus ( QString::null );
src/serverdlg.cpp:                edtCurrentSessionDir->setText ( sessionDir != QString::null ? sessionDir : "" );
src/serverdlg.cpp:        if ( strRecordingDir == QString::null )
src/recorder/jamcontroller.cpp:        bRecorderInitialised = ( strRecorderErrMsg == QString::null );
src/recorder/jamcontroller.cpp:        strRecorderErrMsg    = QString::null;
src/recorder/jamrecorder.cpp: * @return QString::null on success else the failure reason
src/recorder/jamrecorder.cpp:    QString   errmsg = QString::null;
```

Reply:
OK, we can deal with that issue separately then.